### DISCUSSION TITLE
Increasing render distance limit past 32 chunks

### DISCUSSION THREAD

**Discussion by SirPytan** [FabricMC/fabric-api#1587] [Mod Dev Support]
Since there get so many performance improving mods developed for fabric, it would be a shame, if we can't take advantage of the extra frames we get to increase the render distance of minecraft with fabric past the 32 chunks line. A max of 64 chunks would be enough for the beginning, but in general it would be cool to set it to what ever amount your pc can handle.

I have heard that it is hard coded, but it would be nice if a skilled java developer would be able to extend the limit and take care of that. I and probably a lot of people would be very thankful for that.

**Comment by haykam821** [comment]
There is a mod named [Slyde](https://modrinth.com/mod/slyde) that removes all slider limits.

**Reply by DetachedHat1799** [reply]
However, according to what Ive found, minecraft itself has a hard limit of 32 chunks (not sure but I set render distance to 128 and it only kept up to 34 loaded)
